providers/class.two-factor-email.php

```diff
@@ -148,7 +148,7 @@ public function generate_and_email_token( $user ) {
 		$token = $this->generate_token( $user->ID );
 
 		/* translators: %s: site name */
-		$subject = wp_strip_all_tags( sprintf( __( 'Your login confirmation code for %s', 'two-factor' ), get_bloginfo( 'name' ) ) );
+		$subject = wp_strip_all_tags( sprintf( __( 'Your login confirmation code for %s', 'two-factor' ), mb_encode_mimeheader( get_option( 'blogname' ), ENT_QUOTES ) ) );
```

Comment:
What about `html_entity_decode()`? Looks fine if it works as expected though.

Reply:
I'd defer to @ajvillegas on whether that was tried or not. `wp_specialchars_decode` which the upstream uses definitely does _not_ fix it, so I would guess that the more specific function here is likely necessary.